README.md

```diff
@@ -132,21 +132,32 @@ how to operate in advance use cases, use case by use case
 explanation of all available options/configurations that can be used
 
 # Architecture
-![Container architecture](/docs/assets/container-architecture.jpg "Container architecture")
+
+There is mainly 2 stack you can use 1. rest.bitcoin.com techstack 2. Chris Troutner JS tech stack
```

Comment:
There is mainly 2 stack you can use 1. rest.bitcoin.com techstack 2. Chris Troutner JS tech stack

I would say:


Two REST APIs can be run: `rest.bitcoin.com` and `bch-api`.

With probably links to both, so that the use can see what they look like live